_posts/2020-10-04-newpipe-0.20.0.md

```diff
@@ -1,154 +1,176 @@
 excerpt_separator: <!-- more -->
 ---
 
-Hey there! Guess who is back? We are. Admittedly, we're quite a bit late with this release. But there is a _very_ good reason for that. Are you ready for it? Sit down, grab some popcorn and a beverage of your choice. This is going to be long.
+Hey there! Back admittedly a bit late with this release, but for _very_ good reason. Ready? Sit down, grab the virtual popcorn and some hydration.
 <!-- more -->
 
 ## A new unified player
 
-"What in the name of the coronavirus is the _unified player_?" you ask. Well, first of all, that's a very weird expression. Where did you even get that one from... - you know what? Save it. We don't want to know. You just keep on using that, we'll keep on talking about the new player.
+"If the new player is the _unified player_, was the old one un-unified?" you ask. The new player is new, and it is unified.
 
-A new player it indeed is. Up until now, NewPipe had three different players bundled in its code: the main, pop-up and background players. All three had separate code and separate playing queues. When you switched from one player to another, in the background, the old player and its queue closed itself, while a new player was created, along with a new queue, which took over then. If that sounds complicated and cumbersome, that's because it is. Moreover, if you accidentally switched using the player buttons below the thumbnail instead of the ones in the drop down menu inside the player, the stream would start from the beginning instead of carrying on from where you left, and you would lose your (possibly carefully built) queue.
+Until now, NewPipe had a main, pop-up and background -player. All three had separate code and playing queues. Switching from one player to another meant the old player and its queue closed itself, while a new player and queue was created. That sounds complicated and cumbersome, because it is. Moreover, if you accidentally switched using the player buttons below the thumbnail instead of the ones in the drop down menu inside the player, the stream would start anew instead of carrying on from where you left off. Losing your (possibly carefully built) queue was the bonus.
 
-Well, no more! The buttons in the player drop down menu have been removed and the action buttons below the thumbnail work like they're supposed to, so now you're at no risk whatsoever of losing your playing position or queue. Whew!
+No more! There are no buttons in the player drop-down menu anymore, and the action buttons below the thumbnail now only do the job of being action buttons.
+Everyone keeps their position _in the queue_. The utopia is here.
 
-This was made possible by rewriting the code so that all players run on one common service. When you switch from one player to another, this service remains running, preserving the stream data, and the new player just picks up from where the old one left off. Everything remains the same, including the queue. Ah, serendipity.
```

Comment:
OK, we get to the point. The user is full of info about the old system, and finally all of that can be discarded. It sort of works, but taking the reader through all that is not the best way of explaining difficulty.

Reply:
It was necessary to delve into details at least somewhat below surface level to explain why some things have changed to what they are now. They are not immediately obvious, and if the user doesn't know them, they will consider these changes bugs. I can already see this by the number of bug reporters I am directing towards the blog post.

Reply:
Notwithstanding detail, the concept of explaining the old is not interesting. Change is interesting.
Making the distinction between new and old keeps track of what the point is. It stops on an immediate level.

Reply:
Whether interesting or boring, this was necessary, as I've said in the previous comment. So it had to be done.